src/common/httpClient/httpClient.ts

```diff
@@ -94,4 +88,34 @@ export class HttpClient implements BasicHttpClient {
       body: responseBody as TDto,
     };
   }
+
+  private async handleResponse(response: Response, config: HttpRequestConfiguration, shouldParseJson: boolean | null) {
+    if (!shouldParseJson) {
```

Comment:
`shouldParseJson` sollte dann nur in dem json/default Fall ausgewertet werden?

Reply:
Changed the name in the most recent push 👍